### PR TITLE
Add MinMon templates

### LINKED ISSUE #8: Add templates for MinMon
Add templates for MinMon plots including:

- gnorm, last 4 cycles + 7 day average
- single cycle gnorm + 7 day average
- single cycle reduction + 7 day average

Evaluate whether the initial/final plots can be easily done as part of this issue or if they warrant a separate effort.

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
Modify `plotObsMon.py `to support minmon plots and add templates to produce:

- minmon summary plot
- minmon 4 cycle gnorm plot
- minmon single cycle gnorm plot

Please note this PR includes the same changes as #11 .  I messed up and started from it rather than from develop.  So I'll make this a draft and convert it to release if/when #11 is merged.

Resolves #8.

### CODE REVIEW

File: ush/plotObsMon.py [hunk 1]
```diff
@@ -131,7 +132,7 @@ def loadConfig(satname, instrument, plot, cycle_tm, cycle_interval, data_locatio
     data_location = mon_dict.get('data')
 
     sats = mon_dict.get('satellites')
-    for sat in sats:
+    for sat in (sats or []):
```
Comment (CoryMartin-NOAA) [inline ush/plotObsMon.py]: cool, I didn't know about this syntax!

Comment (CoryMartin-NOAA) [reply ush/plotObsMon.py]: I'm wondering though if it is more clear to do something like:
```
if 'satellites' in mon_dict.keys():
   sats = mon_dict.get('satellites')
   for sat in sats:
```
Curious to see what others think about this both here and for the minimization section.

File: ush/plotObsMon.py [hunk 2]
```diff
@@ -148,3 +149,21 @@ def loadConfig(satname, instrument, plot, cycle_tm, cycle_interval, data_locatio
 
                 eva(plot_yaml)
                 os.remove(plot_yaml)
+
+    mins = mon_dict.get('minimization')
+    satname = None
+    instrument = None
+    for min in (mins: or []):
```
Comment (kevindougherty-noaa) [inline ush/plotObsMon.py]: extra `:`?

File: ush/plotObsMon.py (hunk 1 above)
Comment (kevindougherty-noaa) [reply ush/plotObsMon.py]: I agree with Cory, I don't know if I love that syntax and would rather be more explicit with the empty list.

File: ush/plotObsMon.py (hunk 2 above)
Comment (kevindougherty-noaa) [reply ush/plotObsMon.py]: Also, same comment to Cory's point above.

File: ush/plotObsMon.py (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply ush/plotObsMon.py]: Ok, will go with greater clarity.

**Review by CoryMartin-NOAA — COMMENTED** [review]
last comment and otherwise it looks good, perhaps add some comments to say something like "if specified, generate template YAMLs and figures for minimization", etc.?

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**